datafusion/physical-expr/src/equivalence/properties.rs

```diff
@@ -2720,8 +2799,8 @@ mod tests {
                         Arc::clone(&schema3),
                     ),
                 ],
-                // Expected
-                vec![],
+                // Expected: union sort orders
+                vec![vec!["a", "b", "c"]],
```

Comment:
I think these test changes are not correct.

Reply:
I agree -- these changes make the expected output incorrect